operations/helm/charts/mimir-distributed/templates/alertmanager/alertmanager-statefulset.yaml

```diff
@@ -104,6 +109,9 @@ spec:
             {{- range $key, $value := .Values.alertmanager.extraArgs }}
             - "-{{ $key }}={{ $value }}"
             {{- end }}
+            {{ if .Values.alertmanager.fallbackConfig }}
+            - "--alertmanager.configs.fallback=/etc/alertmanager/fallback-config.yaml"
+            {{ end }}
```

Comment:
we can remove these from here since they're in the yaml config
```suggestion
```

Reply:
removed